src/utils/validator.ts

```diff
@@ -4,7 +4,8 @@ import { TypeGuard } from '@sinclair/typebox/guard'
 import { Value } from '@sinclair/typebox/value'
 
 export interface useValidatorOptions {
-  includeAjvFormats: boolean
+  includeAjvFormats: boolean,
```

Comment:
```suggestion
  includeAjvFormats: boolean
```

I'm used to not include trailing comas for interface properties, but I guess it's a developer choice ^^

Reply:
Fixed this - most likely a misconfigured linter issue.